--- disk_usage_original.py	2022-07-12 15:46:44.872700713 +0300
+++ disk_usage_fixed.py	2022-07-12 15:52:24.495491818 +0300
@@ -1,6 +1,7 @@
 #!/usr/bin/env puthon3
 
 import shutil
+import sys
 
 
 def check_disk_usage(disk, min_absolute, min_percent):
@@ -12,9 +13,9 @@
     return True
 
 
-if not check_disk_usage("/", 2*2**30, 10):
+if not check_disk_usage("/", 2, 10):
     print("ERROR! Not enough disk space")
-    return 1
+    sys.exit(1)
 
 print("Everything OK")
-return 0
+sys.exit(0)
